pkg/kubelet/dockertools/manager.go

```diff
@@ -439,6 +440,18 @@ func makeMountBindings(mounts []kubecontainer.Mount, podHasSELinuxLabel bool) (r
 			}
 
 		}
+		// Shared Propagation Mode
+		if m.Shared {
+			if supportSharedPropagationMode {
+				if !m.ReadOnly && !(m.SELinuxRelabel && podHasSELinuxLabel) {
+					bind += ":shared"
+				} else {
+					bind += ",shared"
+				}
+			} else {
+				glog.Warningf("Shared propagation mode is to be supported on docker versions >= v1.10")
```

Comment:
Sorry if I wasn't clear, my intention was to suggest rewording the log message to 'requires docker >= v1.10' instead of 'is to be supported on docker versions >= v1.10'.